Alejandro_Presch/Day-1-Html/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="utf-8">    
+    
+    <title>Encuesta servicio Gigared</title>
+  </head>
+  
+  <body id="body">
+    <header>
+
+        <h1 id="title" align = "center"> Servicio de Internet Gigared</h1>
+
+        <div>
+            <h2 id="titleh2">Encuesta de satisfacción del servicio</h2><br>
+            <h3 id="titleh3">Ingrese los siguientes datos:</h3>
+        </div>
+    </header>   
+  
+    <div>
+      <section>
+        <form>
+              
+          <label for="nombre"> Nombre: </label>
```

Comment:
Esta mal la relación del for, tiene que estar relacionada con un id del input. [Referencia](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)